### PR TITLE
add UI frame for PaxCounter module

### PR DESCRIPTION
**Pull request by thoherr**
## add UI frame for PaxCounter module

- implements #3284 
- additional non related change: had to limit upload speed to 11200 for my board

### CODE REVIEW

File: variants/tlora_v2_1_16/platformio.ini [hunk 1]
```diff
@@ -3,4 +3,5 @@ extends = esp32_base
 board = ttgo-lora32-v21
 build_flags = 
   ${esp32_base.build_flags} -D TLORA_V2_1_16 -I variants/tlora_v2_1_16
-  -DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
\ No newline at end of file
+  -DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+upload_speed = 115200
```
Comment (caveman99) [inline variants/tlora_v2_1_16/platformio.ini:7]: This seems unrelated to functionality?

**Review by caveman99 — APPROVED** [review]

File: variants/tlora_v2_1_16/platformio.ini (hunk 1 above)
Comment (thoherr) [reply variants/tlora_v2_1_16/platformio.ini:7]: Yes, see my second bulletpoint in PR description. That's only neccessary to upload firmware to the board. since some other variants have this settings in their platformio.ini, i included it in the PR

Comment (thebentern) [reply variants/tlora_v2_1_16/platformio.ini:7]: This is not true of every setup though. Some of us have CP2102 based TLoRA v2 1.6 which flash fine at 921600 baud

Comment (thoherr) [reply variants/tlora_v2_1_16/platformio.ini]: Reverted the commit with this change, since it is completely unrelated for the feature and as @thebentern mentioned not valid for all setups

**Review by thebentern — APPROVED** [review]
